tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -249,7 +250,12 @@
 			</patternset>
 		</unzip>
 		<touch file="target/classes/META-INF/javamail.default.address.map"/>
-		
+
+		<unzip src="lib/jline-3.14.1.jar" dest="${class.dir}">
```

Comment:
BSD-3 (https://github.com/jline/jline3/blob/master/LICENSE.txt):

"Redistributions in binary form must reproduce the above copyright
notice, this list of conditions and the following disclaimer
in the documentation and/or other materials provided with
the distribution."

I assume `*.*` satisfies this requirement.